### PR TITLE
Properly install `chromedriver` on Mac.

### PR DESCRIPTION
**Pull request by s1l3ntcat**
This pull request is basically the same as another I did a while back. Basically, all I did is remove what seemed like redundant colours in a script file, and also added a line to properly install `chromedriver` on Mac.

### CODE REVIEW

**Comment by s1l3ntcat** [comment]
For reference, this was the pull request: https://github.com/EFForg/https-everywhere/pull/15476

File: test/script.py [hunk 1]
```diff
@@ -16,14 +16,9 @@
...
 
 class bcolors:
-    HEADER = '\033[95m'
-    OKBLUE = '\033[94m'
     OKGREEN = '\033[92m'
-    WARNING = '\033[93m'
     FAIL = '\033[91m'
     ENDC = '\033[0m'
-    BOLD = '\033[1m'
-    UNDERLINE = '\033[4m'
```
Comment (pipboy96) [inline test/script.py]: I think this should be moved out into a separate PR, it would be merged much faster in this case.

**Comment by pipboy96** [comment]
@s1l3ntcat I rebased your commit onto the current state of the repo. Don't worry, the color variables have been already merged (see https://github.com/EFForg/https-everywhere/blob/master/test/script.py).

**Comment by pipboy96** [comment]
@s1l3ntcat Please avoid creating PRs from `master` branch of your fork in the future, use a separate branch for each PR.

**Comment by pipboy96** [comment]
@zoracon Please review, this package was indeed moved into Homebrew Casks from main repository.
